README.md

```diff
@@ -1,13 +1,8 @@
-# Swift Programming Language Evolution
+# Swift Evolution
```

Comment:
Might want to ask @DougGregor why we included this in the heading originally before changing it.

Reply:
It's fine to change.